### PR TITLE
Update ActiveRateGroup to new constructor and configure() function

### PR DESCRIPTION
**Pull request by timcanham**
| | |
|:---|:---|
|**_Originating Project/Creator_**| JPL Infrastructure |
|**_Affected Component_**| `Svc/ActiveRateGroup` |
|**_Affected Architectures(s)_**| all |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**| y |
|**_Builds Without Errors (y/n)_**| y |
|**_Unit Tests Pass (y/n)_**| y |
|**_Documentation Included (y/n)_**| y |

---
## Change Description

This PR changes the constructor to be compliant with FPP code generation and creates a `configure()` method to provide the context values.

## Rationale

Makes the code compliant and simplifies the FPP.

## Testing/Review Recommendations

Re-run unit tests and Ref app.

## Future Work

None

### CODE REVIEW

**Comment by timcanham** [comment]
Please merge #1467 first. This is based on that branch.

File: RPI/Top/instances.fpp [hunk 1]
```diff
@@ -313,18 +311,13 @@ module RPI {
   instance rateGroupDriverComp: Svc.RateGroupDriver base id 1700 \
   {
 
-    phase Fpp.ToCpp.Phases.configObjects """
+    phase Fpp.ToCpp.Phases.configComponents """
```
Comment (bocchino) [inline RPI/Top/instances.fpp:314]: Make sure it's OK to declare this array inside a function (array is copied before use).

File: RPI/Top/instances.fpp [hunk 2]
```diff
@@ -23,12 +23,11 @@ module RPI {
   {
 
     phase Fpp.ToCpp.Phases.configObjects """
-    NATIVE_UINT_TYPE context[] = { RpiDemo::RG_CONTEXT_10Hz, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+    NATIVE_INT_TYPE context[] = { RpiDemo::RG_CONTEXT_10Hz, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```
Comment (bocchino) [inline RPI/Top/instances.fpp:26]: What is the reason for changing UINT to INT?

**Review by bocchino — COMMENTED** [review]
Looks good, just a couple comments/questions.

File: RPI/Top/instances.fpp (hunk 2 above)
Comment (timcanham) [reply RPI/Top/instances.fpp:26]: I changed the type to INT to get rid of some compile errors when using them in loops.

**Comment by timcanham** [comment]
Closed for #1510

**Pull request closed without merging**